python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -1757,6 +1764,7 @@ def runDataRef(self, patchRefList1, patchRefList2, subdir=""):
         else:
             forced1 = unforced1
             forced2 = unforced2
+
```

Comment:
You shouldn't add lines that aren't related to the stated function of the commit.

Reply:
I think that I added a temporary debugging statement here and left the code in the way that I would have written it, with an extra space. I'll remove.